### PR TITLE
[snmp] review tests

### PR DESCRIPTION
**Pull request by yannmh**

### CODE REVIEW

File: tests/test_snmp.py [hunk 1]
```diff
@@ -14,318 +14,246 @@
...
+        config = {
+            'instances': [self.generate_instance_config(
+                self.SUPPORTED_METRIC_TYPES + self.UNSUPPORTED_METRICS)]
+        }
+        self.run_check_twice(config)
+
+        # Test metrics
+        for metric in self.SUPPORTED_METRIC_TYPES:
+            metric_name = "snmp." + metric['name']
+            self.assertMetric(metric_name, tags=self.CHECK_TAGS, count=1)
```
Comment (LeoCavaille) [inline tests/test_snmp.py]: Should you also ensure that we get none of the `UNSUPPORTED_METRICS`?

**Comment by LeoCavaille** [comment]
Nice! The `force_reload` could be very useful, thanks for adding it!

File: tests/test_snmp.py (hunk 1 above)
Comment (yannmh) [inline tests/test_snmp.py]: Just added it. Thanks

**Comment by LeoCavaille** [comment]
Thanks :+1:

**Pull request merged**